votor/src/consensus_pool/vote_pool.rs

```diff
@@ -1,129 +1,304 @@
+//! Container to store received votes and associated stakes.
+//!
+//! Also implements various checks for slashable behavior and duplicate votes.
```

Comment:
This seems to imply duplicate votes will never be slashable. Both definitions are vague.

You need to say what are "duplicate votes" and what are "slashable behavior" here.